lib/python/ensembl/brc4/runnable/process_gff3.py

```diff
@@ -248,13 +252,15 @@ def simpler_gff3(self, gff3_in: TextIO, out_gff_path: Path, out_funcann_path: Pa
                         feat = self.transcript_gene(feat)
                     elif feat.type == "CDS":
                         feat = self.cds_gene(feat)
+                    elif feat.type in ('mobile_genetic_element', 'transposable_element'):
+                        feat = self.format_mobile_element(feat, functional_annotation)
                     
                     # Normalize the gene structure
                     if feat.type in allowed_gene_types:
                         feat = self.normalize_gene(feat, functional_annotation, fail_types)
-                    else:
+                    elif feat.type not in allowed_non_gene_types:
```

Comment:
If we can think of a future expansion of the `allowed_non_gene_types`, we can go with:
```suggestion
                    elif feat.type in allowed_non_gene_types:
                        pass
                    else:
```